package.json

```diff
@@ -25,14 +25,12 @@
   },
   "scripts": {
     "nx": "nx",
-    "start": "nx serve",
-    "build": "nx build",
+    "build": "nx run-many --target=build --all",
```

Comment:
I would still prefer to leave the default npm tasks to be executed for a single project. I would suggest to use something like `build:all` to run those kind of commands.

Reply:
done in c260e4f2a97136de4b6b2962535e4565f011ec48

Reply:
`build` was not reverted